### DISCUSSION TITLE
Customizing Swell

### DISCUSSION THREAD

**Discussion by unknownsouls** [swellstores/community#490] [Help] [answered]
Hi Swell,

Could you consider creating a tutorial on customizing Swell? Specifically, a guide/video on enhancing product pages within your themes, and linking to the Swell admin, similar to what you have done with the Pages. As it stands, product pages are not editable and lack the option to add sections.

I appreciate the focus on developing the app store and hopefully this would help with integrations. In addition, offering users the freedom to make subtle changes could significantly enhance customization and engagement to set Swell apart, especially as Shopify adopts features like built-in subscriptions. This flexibility could help offer unique storefronts beyond Shopify's sameness.

Please make Swell more accessible so we have more discussions in the `Show and tell' section instead of 'help'

**Comment by ericingram** [comment, marked as answer]
Hi @unknownsouls,

You make a good point and we're in full agreement. In fact our primary project objective today rolls up to this: make it easy to customize and launch a storefront. The App Store is an important part of that because it is the foundation of how our new storefront product works. We made enough progress last year on apps to be able to shift our efforts here. Look out for updates and early previews of the solution in our newsletter this year.

Thanks for the feedback!
